### ISSUE TITLE
Remember Window Size

### ISSUE THREAD

**Issue by proedie** [Swordfish90/cool-retro-term#91]
Hello!

cool-old-term should remember the size (and position and the screen) of its window when the programme is closed. When re-opened, the window should have the same size (and position) as before.

Cheers!
##

**Comment by paulolieuthier** [comment]
Please, implement this future. It's the application's job to remember it's size. It's specially important for a terminal.

**Comment by maandree** [comment]
I disagree and think that it is your window manager that manage your windows, including remembering the size and position if you want that, which I do not.

**Comment by maandree** [comment]
Also it is not “the applications' job”. There is no consensus on how this should be done.

**Comment by Translator5** [comment]
This feature do cause a ugly issue on cinnamon desk. It wont let you size your fullscreen down again.

**Comment by wyan** [comment]
Is this request abandoned? It would be really useful to have new terminals open with the correct size, preferably profile-definable.

**Comment by arisboch** [comment]
Most terminal emulators I used have this option, why not this one, too?

**Comment by depmco** [comment]
Just another vote for opening with either the last size set or 80X24.  It's a drag to always have crt start with e.g. 137x44 and have to resize each time.

**Comment by Dunravin** [comment]
I'm using xfwm4, there's no obvious solution to prevent CRT from opening on my left most monitor. 

xfwm4 has a "smart placement" feature found in window manager tweaks application but CRT does not conform in anyway to trigger the smart placement which works according to window size, in fact there isn't any combination of window size and slider setting that triggers placement for CRT so I would argue "leaving it up to the window manager" is not a viable solution if CRT is not reporting sufficient detail to the window manager for the window manager to manage the CRT window to begin with.

Other terminals are triggering smart placement as are other applications which leads me to believe this is a CRT specific quirk so clearly there is something more that could be done to bring into line with other terminals.

I have had some success with devilspie2 in placing and sizing the CRT window but need to further refine its configuration and to get that working automatically at login but I feel this should really not be necessary and the CRT application should be capable of at least remembering it's last size/position when closed.

**Comment by NeonTropics** [comment]
*resurrection time* - Any updates on implementing a passable switch or profile linked setting that controls the window geometry on launch? Xfce likes to start it covering 85% of the screen... every time... regardless of the size of the window when you exit.
